src/pages/index.jsx

```diff
@@ -97,6 +98,8 @@ export default function Index({ title }) {
 
           <MiddleSectionsContainer>{/* <Stats /> */}</MiddleSectionsContainer>
           <Separator />
+          <Workshops />
+          <Separator />
```

Comment:
the workshops component should go in the `MiddleSectionsContainer` after `Stats`! my bad for the terrible naming 😅 it won't need a separator either since there's one after the container

```suggestion
          <MiddleSectionsContainer>
            {/* <Stats /> */}
            <Workshops />
          </MiddleSectionsContainer>
          <Separator />
```

Reply:
can you change the `MiddleSectionsContainer` to this? it should enable the absolute positioning of the workshop
```suggestion
          const MiddleSectionsContainer = styled.div`
		  position: relative;
		  background-image: url(/assets/images/middle_background.jpg);
		  background-size: contain;
		  background-repeat: no-repeat;
		  background-position: center;
		  object-fit: cover;
		
		  width: 100%;
		  aspect-ratio: 1280/2303;
	`
```